libs/ropes/src/Ropes/Twilio.hs

```diff
@@ -129,7 +129,6 @@ data CarrierInfo = CarrierInfo
 data PhoneType
   = Landline
   | Mobile
-  | VoIp
```

Comment:
I'm unfortunately unsure how twilio relates to our SNS sending code :thinking:

Reply:
This seems to be entirely unrelated - the Twilio `PhoneType` is something entirely different, this PR shouldn't be touching this.